Dockerfile

```diff
@@ -1,6 +1,6 @@
-# Docker image for the Drone AWS Opsworks plugin
+# Docker image for the Drone Opsworks plugin
 #
-#     cd $GOPATH/src/github.com/drone-plugins/drone-aws-opsworks
+#     cd $GOPATH/src/github.com/drone-plugins/drone-opsworks
 #     make deps build docker
 
 FROM alpine:3.2
```

Comment:
Do we have a standard alpine version for plugins?  Just wondering because I have been moving some to `alpine:3.3`  Should we do that here as well?